### PR TITLE
Update slack page

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Update slack page with simpler content and more prominent CTA button
| Fixed ticket | ~

Screenshot:

Screenshot 2022-08-04 at 19 38 39

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
@eternoendless OK for me but I don't merge as I don't know if you want someone else review (Krystian? Patrick?)

**Comment by kpodemski** [comment]
Looks good. I've noticed one thing. Can we remove text-capitalize? :D

**Comment by eternoendless** [comment]
> Looks good. I've noticed one thing. Can we remove text-capitalize? :D

Yeah I found that annoying as well. It's done via CSS (see https://github.com/PrestaShop/ps-org-theme/blob/0c653b2b8d0d95850ee2c4c855ff632163f72c9f/assets/scss/_buttons.scss#L6). How about we remove it in a follow-up PR?

**Comment by coffrep** [comment]
Looks very nice. Just one suggestion in the first paragraph with "a place to be". Could this be changed to "the place to be"? This would be better if you can change, [more information here in the Merriam Webster](https://www.merriam-webster.com/dictionary/the%20place%20to%20be).

File: content/slack/_index.md [hunk 1]
```diff
@@ -1,70 +1,37 @@
...
 
 aliases:
   - /slack/readme/
 ---
 
-# About the project's Slack
+# PrestaShop Project's Slack chat
 
-## General presentation
+The project's Slack chat is open to anyone who wants to discuss, help and contribute to the PrestaShop open source project. If you are an active contributor, or want to become one, this is definitely a place to be!
```
Comment (coffrep) [inline content/slack/_index.md:15]:
```suggestion
The project's Slack chat is open to anyone who wants to discuss, help and contribute to the PrestaShop open source project. If you are an active contributor, or want to become one, this is definitely the place to be!
```

Comment (coffrep) [reply content/slack/_index.md:15]: Just one suggestion in the first paragraph: "the place to be" could be better, [more information here in the Merriam Webster](https://www.merriam-webster.com/dictionary/the%20place%20to%20be)

File: content/slack/_index.md [hunk 2]
```diff
@@ -1,70 +1,37 @@
...
     parent: 'Resources'
-    name: 'Slack'
+    name: 'Slack chat'
 
 aliases:
   - /slack/readme/
 ---
 
-# About the project's Slack
+# PrestaShop Project's Slack chat
```
Comment (coffrep) [inline content/slack/_index.md:13]:
```suggestion
# PrestaShop Project's Slack Chat
```

File: content/slack/_index.md [hunk 3]
```diff
@@ -1,70 +1,37 @@
 ---
-title: About the project's Slack
+title: PrestaShop Project's Slack chat
```
Comment (coffrep) [inline content/slack/_index.md:2]:
```suggestion
title: PrestaShop Project's Slack Chat
```

File: content/slack/_index.md (hunk 2 above)
Comment (coffrep) [reply content/slack/_index.md:13]: Uppercasing all words in this title

**Review by coffrep — CHANGES_REQUESTED** [review]

File: content/slack/_index.md (hunk 1 above)
Comment (eternoendless) [reply content/slack/_index.md:15]: You're absolutely right, no idea how I missed that!

**Review by coffrep — APPROVED** [review]

**Comment by eternoendless** [comment]
Thanks for the review!

**Pull request merged**